EIPS/eip-draft_increase_blob_base_fee.md

```diff
@@ -0,0 +1,67 @@
+---
+eip: <to be assigned>
```

Comment:
```suggestion
eip: 7762
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.